src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs

```diff
@@ -54,7 +54,7 @@ internal class RazorSemanticTokensLegend
         public static int CSharpString => TokenTypesLegend["string"];
         public static int CSharpPunctuation => TokenTypesLegend["punctuation"];
 
-        private static readonly IReadOnlyCollection<string> _tokenTypes = new string[] {
+        public static readonly IReadOnlyCollection<string> TokenTypes = new string[] {
```

Comment:
I made this `public` so that we could make the SemanticTokens Baseline more readable.